content/en/docs/tasks/observability/metrics/telemetry-api/test.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=none
+echo "$snip_before_you_begin_1" | istioctl install --set hub=gcr.io/istio-testing -y -f -
```

Comment:
@frankbu : is it okay to have it this way, or should we change the main page to include the istioctl command so that snips can be directly used?

Reply:
update to use ${HUB} and ${TAG}

Reply:
Most docs would include the istioctl install instruction in the doc (snip), but this one is just showing the yaml, which is also OK IMO. So installing it in the test with `echo | istioctl install` seems like a good way to test the snip.